packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -407,16 +407,11 @@ function createApp(
     ? ` --interactive=true`
     : ` --interactive=false`;
 
-  console.log(
-    `new ${args} --preset="${preset}"${appNameArg}${styleArg}${interactiveArg} --collection=@nrwl/workspace`
-  );
+  const command = `new ${
```

Comment:
This would break flags like `--skip-install` and `--no-commit`. I believe the proper thing to do is to change how we define `args` above. It filters `process.argv` instead of constructs it from the `parsedArgs`. What do you think?

Reply:
You are right, I did not take that into account.

I'm going to see if I can construct the `args` rather than filter them.

Is this `cli.command === 'tao'` a valid way to decide which CLI we are using?

Reply:
Yes it is